code/obj/machinery/firealarm.dm

```diff
@@ -33,7 +34,7 @@
 
 	if(!net_id)
 		net_id = generate_net_id(src)
-
+	secondary_tick = rand(0, 3)
```

Comment:
I don't like this, rubs me the wrong way.

Reply:
I feel like this is the simplest and most straightforward way to do this? Do you have any suggestions on how I could change it around to be less janky?

Reply:
using the variable machine loop speed subscription service

Reply:
Could you send me either an example or just the proc I need to call? I'm a bit confused on how to do that.